Openai-codex-best-practices-2026.html

```
@@ -0,0 +1,2221 @@
+        h1.title {
+          font-size: 26px;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div id="progress"></div>
+
+    <div class="topbar">
+      <span class="brand">CODEX / BEST PRACTICES</span>
+      <button id="navToggle" aria-label="目次を開く">☰ 目次</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)